### DISCUSSION TITLE
invalid query parameter, yes has to be true or false

### DISCUSSION THREAD

**Discussion by bighb69738** [edgexfoundry/community#214] [Q&A] [answered]
After the command is executed, the following error message appears:

level=ERROR ts=2023-11-27T02:04:45.784550087Z app=core-command source=http.go:47 X-Correlation-ID=a9b7866d-cb5b-432a-afa0-fadc0f6addf1 msg="invalid query parameter, yes has to be true or false"
level=ERROR ts=2023-11-27T02:04:59.671497315Z app=core-command source=http.go:47 X-Correlation-ID=4c8a5db4-be40-4432-b39e-267caefbbdfb msg="invalid query parameter, yes has to be true or false"

How to fix it?

**Comment by jiekechoo** [comment, marked as answer]
@bighb69738 
Hi, you could create an issue in the [core-command](/edgexfoundry/edgex-go/issues) repo, and provide more infomation.

**Reply by bighb69738** [reply]
Yes, I have posted it https://github.com/edgexfoundry/edgex-go/issues/4728
